apps/contrib/templates/matomo/tracking_code.html

```diff
@@ -0,0 +1,4 @@
+{% load static %}
+<!-- Matomo -->
+<script id="matomo" data-cookieDisabled={{ cookie_disabled }} data-url="{{ url }}" data-id="{{ id }}"async src="{% static 'matomo.js' %}"></script>
```

Comment:
feels a bit hacky to put the data attributes on the script tag but worked for me in chrome and firefox. Should be tested in safari as well

Reply:
@philli-m or @khamui Can you test this also in safari, please?